scripts/update/dbMigrations/VariablesStorage_v1.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
if that file is static, could you change it, maybe to:
`scripts/update/v6.1.0/VariablesStorage_v1.php` - it force us do not rewrite it

Reply:
or dbMigrations/VariableStorage_v6.1.0 something that shows us the specific version